src/datasets/features/features.py

```diff
@@ -1427,11 +1437,12 @@ def generate_from_arrow_type(pa_type: pa.DataType) -> FeatureType:
         return {field.name: generate_from_arrow_type(field.type) for field in pa_type}
     elif isinstance(pa_type, pa.FixedSizeListType):
         return Sequence(feature=generate_from_arrow_type(pa_type.value_type), length=pa_type.list_size)
-    elif isinstance(pa_type, pa.ListType):
+    elif isinstance(pa_type, (pa.ListType, pa.LargeListType)):
         feature = generate_from_arrow_type(pa_type.value_type)
         if isinstance(feature, (dict, tuple, list)):
             return [feature]
```

Comment:
Arf I didn't think of how to support large lists of dicts in this case ?

It's maybe not the most convenient to add `large=` to `Sequence` and its weird behavior coming from TFDS. Alternatively we could define a LargeList type. It would also make the YAML simpler:

```
features:
- name: column
  large_list: string
```
instead of 
```
features:
- name: column
  sequence:
    dtype: string
    large: true
```

WDYT ?

Reply:
It was your idea to create a large attribute in Sequence... :sweat_smile: 
https://github.com/huggingface/datasets/pull/4800#discussion_r994396651

Reply:
Yes I completely missed this issue with sequence of dicts sorry -_-
I can take care of making the change this afternoon if you want

Reply:
No, no, I will take care of that...